docs/managing-data-quality/failed-events/failed-events-in-athena-and-bigquery/index.md

```diff
@@ -4,81 +4,54 @@ date: "2020-04-30"
 
 ```sql
 CREATE DATABASE IF NOT EXISTS {{ DATABASE }}
 ```
 
-Then run each sql statement provided in the [badrows-tables repo](https://github.com/snowplow-incubator/snowplow-badrows-tables/tree/master/athena) by copying them into the Athena query editor:
-
-- ![](images/athena-create-table.png)
-    
+Then run each sql statement provided in the [badrows-tables repository](https://github.com/snowplow-incubator/snowplow-badrows-tables/tree/master/athena) by copying them into the Athena query editor. We recommend creating all tables, although you can skip the ones you are not interested in.
 
-Create table in Athena
+:::info Placeholders
 
-As example of using your Athena tables, you might start by getting counts of each failed event type from the last week. Repeat this query for each table you have created:
+Note that the sql statements contain a few placeholders which you will need to edit before you can create the tables:
 
-```sql
-SELECT COUNT(*) FROM schema_violations
-WHERE from_iso8601_timestamp(data.failure.timestamp) > DATE_ADD('day', -7, now())
-```
+* `{{ DATABASE }}` — as above, change this to the name of your pipeline, e.g. `prod1` or `qa1`.
+* `s3://{{ BUCKET }}/{{ PIPELINE }}` — this should point to the directory in S3 where your bad rows files are stored.
 
-- ![](images/athena-count.png)
-    
+:::
 
-Athena query
+![Creating a table in Athena](images/athena-create-table.png)
 
-If you have schema violations, you might want to find which tracker sent the event:
+  </TabItem>
+  <TabItem value="bigquery" label="BigQuery">
 
-```sql
-SELECT data.payload.enriched.app_id, COUNT(*) FROM schema_violations
-WHERE from_iso8601_timestamp(data.failure.timestamp) > DATE_ADD('day', -7, now())
-GROUP BY data.payload.enriched.app_id
-```
-
-You can do a deeper dive into the error messages to get a explanation of the last 10 failures:
-
-```sql
-SELECT data.failure.messages[1].field AS field,
-       data.failure.messages[1].value AS value,
-       data.failure.messages[1].error AS error,
-       data.failure.messages[1].json AS json,
-       data.failure.messages[1].schemaKey AS schemaKey,
-       data.failure.messages[1].schemaCriterion AS schemaCriterion
-FROM schema_violations
-ORDER BY data.failure.timestamp DESC
-LIMIT 10
-```
-
-## BigQuery instructions
+:::note
 
 These instructions make use of the [bq command-line tool](https://cloud.google.com/bigquery/docs/bq-command-line-tool) which is packaged with the [google cloud sdk](https://cloud.google.com/sdk/docs). Follow the sdk instructions for how to [initialize and authenticate the sdk](https://cloud.google.com/sdk/docs/initializing). Also take a look at the [BigQuery dashboard](https://console.cloud.google.com/bigquery) as you run these commands, so you can see your tables as you create them.
 
-##### Missing fields ⚠️
```

Comment:
Removing thanks to https://github.com/snowplow-incubator/snowplow-badrows-tables/pull/14